<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>To Do List</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/todolisticon.png</normaloff>images/todolisticon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tasks">
       <attribute name="title">
        <string>tasks</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>29</x>
          <y>90</y>
          <width>731</width>
          <height>411</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
         <property name="spacing">
          <number>2</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <property name="sizeConstraint">
            <enum>QLayout::SetFixedSize</enum>
           </property>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLineEdit" name="tb_title">
             <property name="placeholderText">
              <string>ex: Do homework</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tb_description">
             <property name="placeholderText">
              <string>ex: page 14 to16</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="tb_time">
             <property name="placeholderText">
              <string>ex: 11 july at 20 o'clock</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="add_btn">
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>721</width>
          <height>61</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Condensed</family>
          <pointsize>33</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="text">
         <string>TO DO LIST</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="complete">
       <attribute name="title">
        <string>complete</string>
       </attribute>
       <widget class="QWidget" name="gridLayoutWidget">
        <property name="geometry">
         <rect>
          <x>59</x>
          <y>110</y>
          <width>671</width>
          <height>361</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_comp"/>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>20</y>
          <width>671</width>
          <height>81</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Roboto Condensed</family>
          <pointsize>33</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Completed Tasks</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
